src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const LONG_TERM = 7;
+  const MEDIUM_TERM = 3;
+  const LONG_TERM_DISCOUNT = 50;
+  const MEDIUM_TERM_DISCOUNT = 20;
+  const BASE_PRICE = 40;
+
+  const price = days * BASE_PRICE;
+
+  if (days >= LONG_TERM) {
+    return price - LONG_TERM_DISCOUNT;
+  } else if (days >= MEDIUM_TERM) {
```

Comment:
```suggestion
  } 
  
  if (days >= MEDIUM_TERM) {
```
<img width="872" alt="image" src="https://github.com/mate-academy/js_task-transportation-on-vacation/assets/70220667/d164aa38-2b73-4f37-88b0-be19fca540ac">